OpenEdXMobile/AndroidManifest.xml

```diff
@@ -274,6 +274,9 @@
             android:name="com.google.android.gms.version"
             android:value="@integer/google_play_services_version" />
 
+        <!-- Setup Firebase -->
+        <meta-data android:name="firebase_analytics_collection_enabled" android:value="${firebaseEnabled}" />
```

Comment:
I guess it would be better to have Facebook grouped together with Google since they're both part of the social services group, and the Firebase entry moved after them.